fixtures/hasura/chinook/metadata/chinook.hml

```diff
@@ -21,11 +21,25 @@ definition:
           comparison_operators:
             _eq:
               type: equal
+            _in:
+              type: custom
```

Comment:
This isn't quite right. `_in` should be using the standard `in` typed comparison operator: `ComparisonOperatorDefinition::In`, not a custom operator.

Reply:
Good point! I've updated the PR